test/unit/core/simd/detail/builtin_simd_test.cpp

```diff
@@ -145,6 +145,10 @@ TEST(builtin_simd, simd_traits)
     EXPECT_TRUE((std::is_same_v<seqan3::simd::simd_traits<int32x4_t>::swizzle_type, uint8x16_t>));
     EXPECT_TRUE((std::is_same_v<seqan3::simd::simd_traits<int64x2_t>::swizzle_type, uint8x16_t>));
 
+    // EXPECT_TRUE((std::is_same_v<seqan3::simd::simd_traits<int16x8_t>::rebind_type, uint8x16_t>));
```

Comment:
```suggestion
    EXPECT_TRUE((std::is_same_v<seqan3::simd::simd_traits<int16x16_t>::rebind_type<uint16_t>, uint16x16_t>));
```

And some more if you find matching types that only differ in their signedness.